instapi/models/feed.py

```diff
@@ -80,24 +143,64 @@ def iter_resources(self, *, video: bool = True, image: bool = True) -> Iterable[
                 yield Image.create(media['image_versions2']['candidates'][0])
 
     def resources(self, video: bool = True, image: bool = True, limit: Optional[int] = None) -> List[Resources]:
+        """
+        Generate list of images/videos, which contains in the post
+
+        :param video: true - add videos, false - ignore videos
+        :param image: true - add images, false - ignore images
+        :param limit: number of images/videos, which will be added to the list
+        :return: list with images/videos
+        """
         return to_list(self.iter_resources(video=video, image=image), limit=limit)
 
     def iter_videos(self) -> Iterable['Video']:
+        """
+        Create generator for iteration over videos, which contains in the post
+
+        :return: generator with videos, which contains in the post
+        """
         return cast(Iterable['Video'], self.iter_resources(video=True, image=False))
 
     def videos(self, limit: Optional[int] = None) -> List['Video']:
+        """
+        Generate list of videos, which contains in the post
+
+        :param limit: number of videos, which will be added to the list
+        :return: list with videos
+        """
         return to_list(self.iter_videos(), limit=limit)
 
     def iter_images(self) -> Iterable['Image']:
+        """
+        Create generator for iteration over images, which contains in the post
+
+        :return: generator with images, which contains in the post
+        """
         return cast(Iterable['Image'], self.iter_resources(video=False, image=True))
 
     def images(self, limit: Optional[int] = None) -> List['Image']:
+        """
+        Generate list of images, which contains in the post
+
+        :param limit: number of images, which will be added to the list
+        :return: list with images
+        """
         return to_list(self.iter_images(), limit=limit)
 
     def image(self) -> Optional['Image']:
+        """
+        Return next image from post
```

Comment:
Please change the docstring.

`image` method returns the first image from feed or None 
in a case when feed doesn't contains images.